pkg/tracing/tracing.go

```diff
@@ -148,9 +155,9 @@ func LogEventf(r *http.Request, format string, args ...interface{}) {
 	}
 }
 
-// StartSpan starts a new span from the one in the request context
-func StartSpan(r *http.Request, operationName string, spanKind ext.SpanKindEnum, opts ...opentracing.StartSpanOption) (opentracing.Span, *http.Request, func()) {
-	span, ctx := opentracing.StartSpanFromContext(r.Context(), operationName, opts...)
+// StartSpan starts a new span from the span in provided context
```

Comment:
Maybe better to use `StartSpan starts a new span from the one in provided context and the request.`

Reply:
`StartSpan` now starts span from provided context and connects it to provided request. Updated message